documentation/modules/ROOT/pages/configuration/signalling.adoc

```diff
@@ -231,9 +243,21 @@ Currently {prodname} supports the `incremental` and `blocking` types.
 | An array of comma-separated regular expressions that match the fully-qualified names of the tables to include in the snapshot. +
 Specify the names by using the same format as is required for the xref:{context}-property-signal-data-collection[signal.data.collection] configuration option.
 
-|`additional-condition`
+|`[.line-through]#additional-condition#`
+|_N/A_
+| An optional string that specifies a condition that the connector evaluates to designate a subset of records to include in a snapshot. +
+ +
+  [NOTE]
+  ====
+  This property is deprecated and should be replaced by additional-conditions.
+  ====
+
+|`additional-conditions`
 |_N/A_
-| An optional string that specifies a condition that the connector evaluates to designate a subset of records to include in a snapshot.
+| An optional array of additional condition that specifies a condition that the connector evaluates to designate a subset of records to include in a snapshot. +
+Each additional condition is an object with `data-collection` and `filter` properties. You can now specify different filters for different data collection. +
+* The `data-collection` property is the fully-qualified name of the data collection for which the filter will be applied.
```

Comment:
Also this bullet list is not working. Any idea?

Reply:
Same solution as above